pkg/app/ops/firestoreindexensurer/indexes.go

```diff
@@ -53,20 +52,11 @@ func (idx *index) validate() error {
 }
 
 // id builds a unique string based on its fields.
-func (idx *index) id() string {
+func (idx index) id() string {
```

Comment:
nit: I know it's for safety as we talked about before. But it doesn't change itself anymore, or rather, it causes memory copy multiple times. So I'm guessing we'd better keep `*index` as is. But this problem is tiny one; up to you

Reply:
I get your point 👍 Addressed on https://github.com/pipe-cd/pipe/pull/1909/commits/9577184c51d8598151ae109ec45405b08186fe7a 🙏